test/cache_test.rb

```diff
@@ -34,6 +34,7 @@ class UncachedAuthor < Author
     end
 
     class Article < ::Model
+      attributes :title
       # To confirm error is raised when cache_key is not set and cache_key option not passed to cache
       undef_method :cache_key
```

Comment:
seems like we want to make  a `NonCachingModel` poro base class for cases like this.